### DISCUSSION TITLE
Prepend relation using "+someRelation" in the API

### DISCUSSION THREAD

**Discussion by abrman** [pocketbase/pocketbase#1786] [Q&A]
I realize I may be late since this might have been easier to consider before 0.11.0 release, but I'd find ability to prepend a relation using `{ "+someRelation": [relationRecord.id] }` similar to how you can append using `"someRelation+"` quite useful. 

While I was able to implement functionality by reversing the array in the front-end of my app, I wanted to open the discussion on this feature request. While it's possible to work around the limitation with the array reverse, over-cluttering the API isn't a win either. Would this be something good to consider implementing?

**Comment by ganigeorgiev** [comment]
Hm, it could be added but it may cause some confusion since it will differ with the modifiers for the other fields. This also will require a small breaking change in the `PrepareValueWithModifier()` since we'll have to indicate whether it is prefix or suffix and eventually apply some conditional behavior based on the state.

I'll have to think a little more on it but for now remains unplanned since I'm not sure whether it is worth complicating the already messy API and tests.
